### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by hbadi** [trungleduc/jupyterview#39] [Ideas]
Hello,

Here some suggestions :

- It'd be gread to have an update dataset button (for example placed before "previous") 
- modify the number of table values (1 to 256)
- do you plan to add some vtk filters (clip, slice, glyph, iso...) ?

Best regards.

**Comment by trungleduc** [comment]
> * It'd be gread to have an update dataset button (for example placed before "previous")

Can you explain what does it do this button?

> * modify the number of table values (1 to 256)
 
Which table you are talking about?

> * do you plan to add some vtk filters (clip, slice, glyph, iso...) ?

Yes, clip, slice, and iso are planned

**Reply by hbadi** [reply]
- The update dataset button is for refreshing the dataset when the files is modified (its often occurs after a simulation)
- table value number is related to the color lookup table, it's an useful trick in order to get an isosurface display.

**Comment by hbadi** [comment]
Hi,
Do you think possible to develop jupyterview for VS code ?

**Reply by trungleduc** [reply]
> Hi, Do you think possible to develop jupyterview for VS code ?

it's technically possible but I don't have time to do this.
